protocol-tests/src/test/java/fixtures/paging/PagingTests.java

```diff
@@ -0,0 +1,43 @@
+package fixtures.paging;
+
+import com.azure.core.http.rest.PagedIterable;
+import com.azure.core.util.BinaryData;
+import org.junit.Assert;
```

Comment:
`PagingTests.java` in `azure-tests` uses Junit 5. `protocal-tests` project defines
```
    <dependency>
      <groupId>junit</groupId>
      <artifactId>junit</artifactId>
      <version>4.13.1</version>
      <scope>test</scope>
    </dependency>
```

Reply:
Feel free to update protocol-tests to use junit 5

Reply:
Done